odoo80/Dockerfile

```diff
@@ -1,51 +1,5 @@
-FROM vauxoo/docker-ubuntu-base
+FROM vauxoo/docker-ubuntu-base:latest
 MAINTAINER Tulio Ruiz <tulio@vauxoo.com>
 
-RUN wget -qO- https://deb.nodesource.com/setup | sudo bash - \
-    && apt-get upgrade -q \
-    && apt-get install --allow-unauthenticated -q \
-    antiword \
-    bash-completion \
-    libfreetype6-dev \
-    libgeoip-dev \
-    libjpeg-dev \
-    libldap2-dev \
-    libqrencode-dev \
-    libxml2-dev \
-    libxslt1-dev  \
-    libyaml-dev \
-    M2Crypto \
-    nodejs \
-    phantomjs \
-    poppler-utils \
-    python-imaging \
-    python-libxml2 \
-    python-lxml \
-    python-pandas \
-    python-qrcode \
-    ttf-dejavu \
-    xmlstarlet \
-    xsltproc \
-    zlib1g-dev \
-    && npm -g install npm@latest \
-    && npm install -g less \
-    && npm install -g less-plugin-clean-css \
-    && pip install --upgrade pyyaml pillow GeoIP SOAPpy pyopenssl suds xmltodict recaptcha-client egenix-mx-base git+https://github.com/Vauxoo/panama-dv.git \
-    && cd /tmp && git clone --depth=1 https://github.com/thewtex/sphinx-contrib.git \
```

Comment:
I [changed this](https://github.com/Vauxoo/docker-odoo-image/pull/104/commits/7b4fdbcfb35a9578b888154c04acadfbbd8bb007#diff-47daa48a95db7533add915692de62084R43) with the mercurial repo, which is up to date. The last commit on this repo was 3 years ago.